ceph.spec.in

```diff
@@ -33,10 +33,12 @@
 %bcond_with selinux
 %endif
 
-# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12+ and openSUSE TW
+# LTTng-UST enabled on Fedora, RHEL 6+, and SLES 12+ (not openSUSE)
```

Comment:
if I am removing the "+" then that do not match your proposal to only test suse_version and not its value.